packages/carbon-web-components/src/components/structured-list/structured-list.scss

```
@@ -47,32 +52,84 @@ $css--plex: true !default;
   @extend .#{$prefix}--structured-list-row;
 }
 
-:host(#{$prefix}-structured-list-row:hover) .#{$prefix}--structured-list-svg {
-  // TODO: audit
-  fill: $gray-40;
+:host(#{$prefix}-structured-list-row) {
+  ::slotted(#{$prefix}-structured-list-cell) {
+    @extend .#{$prefix}--structured-list-td;
+  }
+
+  ::slotted(#{$prefix}-structured-list-cell:first-of-type) {
+    padding-left: $spacing-05;
+  }
+
+  ::slotted(#{$prefix}-structured-list-cell[nowrap]) {
+    @extend .#{$prefix}--structured-list-content--nowrap;
+  }
 }
 
-:host(#{$prefix}-structured-list-row)
-  ::slotted(#{$prefix}-structured-list-cell) {
-  @extend .#{$prefix}--structured-list-td;
+// selection
+:host(#{$prefix}-structured-list-row[selection-value]:hover) {
+  border-bottom: 1px solid $background-hover;
+  background-color: $background-hover;
+  cursor: pointer;
 }
 
-:host(#{$prefix}-structured-list-row)
-  ::slotted(#{$prefix}-structured-list-cell[nowrap]) {
-  @extend .#{$prefix}--structured-list-content--nowrap;
+:host(#{$prefix}-structured-list-row[selected]) {
+  background-color: $background-selected;
+
+  .#{$prefix}--structured-list-input:checked
+    + .#{$prefix}--structured-list-cell
+    .#{$prefix}--structured-list-svg {
+    fill: $text-primary;
+  }
 }
 
-// "Selected" style with `.#{$prefix}--structured-list-td` does not work somehow - Need investigation
-.#{$prefix}--structured-list-input:checked
-  + .#{$prefix}--structured-list-cell
-  .#{$prefix}--structured-list-svg {
-  fill: $text-primary;
+// skeleton
+:host(#{$prefix}-structured-list-header-cell-skeleton) {
+  &:first-of-type {
+    width: 8%;
+  }
+
+  &:nth-of-type(3n + 2) {
+    width: 30%;
+  }
+
+  &:nth-of-type(3n + 3) {
+    width: 15%;
+  }
+
+  span {
+    @include skeleton;
+
+    width: 75%;
+    height: 1rem;
```

```suggestion
    height: $spacing-05;
```
I know this isn't technically spacing, but maybe this should replaced too?